### DISCUSSION TITLE
Global Exception Handler

### DISCUSSION THREAD

**Discussion by kirtimandwade20** [NeoSOFT-Technologies/rest-java-spring-boot#24] [Ideas]
Spring boot allows us to provide customized global exception handling where we need not to add try catch block everwhere
we can create a separate class for handling exceptions and it also separates the exception handling code from businesss logic code.
Reference:
https://www.baeldung.com/exception-handling-for-rest-with-spring
Spring boot exception handling with AOP
https://javatechonline.com/spring-boot-errors-and-aop-annotations-with-examples/

**Comment by Yogendra-Amarji** [comment]
i am using  Custom HandlerExceptionResolver
